corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from lxml import etree as ET
+
+from django.core.management import BaseCommand
+
+from corehq.apps.app_manager.models import Application, PreloadAction
+from corehq.apps.app_manager.util import save_xform
+from corehq.apps.app_manager.xform import XForm
+
+
+logger = logging.getLogger('app_migration')
+logger.setLevel('DEBUG')
+
+
+class Command(BaseCommand):
+    help = "Migrate an app from case management in the app builder to form builder"
+
+    def handle(self, *args, **options):
+        app_id = args[0]
+        self.options = options
+        logger.info('migrating app'.format(app_id))
+        self.migrate_app(app_id)
+        logger.info('done')
+
+    def migrate_app(self, app_id):
+        app = Application.get(app_id)
+        modules = [m for m in app.modules if m.module_type == 'basic']
```

Comment:
agree that advanced & shadow modules should be skipped, but what about careplan modules?